entrypoint.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+poetry run gunicorn wsgi --bind 0.0.0.0:5000
```

Comment:
You didn't need a separate wsgi file, you could specify `gunicorn "todo_app.app:create_app()"`
The quotation marks prevent the brackets being interpreted by bash as special characters.